testing/endtoend/components/lighthouse_beacon.go

```diff
@@ -182,7 +182,7 @@ func (node *LighthouseBeaconNode) Start(ctx context.Context) error {
 		fmt.Sprintf("--enr-tcp-port=%d", e2e.TestParams.Ports.LighthouseBeaconNodeP2PPort+index),
 		fmt.Sprintf("--port=%d", e2e.TestParams.Ports.LighthouseBeaconNodeP2PPort+index),
 		fmt.Sprintf("--http-port=%d", e2e.TestParams.Ports.LighthouseBeaconNodeHTTPPort+index),
-		fmt.Sprintf("--target-peers=%d", 10),
+		fmt.Sprintf("--target-peers=%d", e2e.TestParams.LighthouseBeaconNodeCount),
```

Comment:
Why was this changed ?

Reply:
It errors on peer discovery, so I thought it should be the amount we choose